test/test_gpio.c

```diff
@@ -12,12 +12,12 @@ void teardown_test(void) {}
 
 // Tests that the GPIO init works. This should simply return true 100% of time unless the test is on
 // x86 and the configuration is incorrect in which case fix it!
-void test_gpio_valid_init(void) { TEST_ASSERT_TRUE(gpio_init()); }
+void test_gpio_init_valid(void) { TEST_ASSERT_TRUE(gpio_init()); }
```

Comment:
Ack. I already merged will clean this up next PR touching this file. Already set my clang-format to avoid issue in future.